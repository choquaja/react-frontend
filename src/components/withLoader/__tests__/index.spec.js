// import React from 'react';
// import {shallow} from 'enzyme';
// import loadingHoc from '../';
// import LoadingIndicator from '../../LoadingIndicator'

// describe('loadingHoc', ()=>{

//  it('Passes down size prop',()=>{
//    const component = shallow(<loadingHoc size={50}/>);
//    expect(LoadingIndicator.props.size).toEqual(50);
// });

// });
