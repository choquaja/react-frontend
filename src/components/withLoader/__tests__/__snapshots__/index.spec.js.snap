// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`loadingHoc Passes down size prop 1`] = `<loadingHoc />`;
